lib/resources/nginx.rb

```diff
@@ -0,0 +1,108 @@
+# encoding: utf-8
+# copyright: 2017
+# author: Aaron Lippold, lippold@gmail.com
+# author: Rony Xavier, rx294@gmail.com
+# license: All rights reserved
+
+require 'pathname'
+
+module Inspec::Resources
+  class Nginx < Inspec.resource(1)
+    name 'nginx'
+    desc 'Use the nginx InSpec audit resource to test information about your NGINX instance.'
+    example "
+      describe nginx.pid_file do
+        its('owner') { should be 'root' }
+      end
+      describe nginx.version do
+        it { should_be >= 1.0.0 }
+      end
+      describe nginx do
+        its('group') { should be 'root' }
+      end
+    "
+
+    def initialize
```

Comment:
Should probably accept an optional parameter of where the `nginx` binary can be found as it may not be in the InSpec user's path.